source/renderer/app/components/wallet/transactions/WalletTransactionsHeader.scss

```diff
@@ -0,0 +1,43 @@
+.component {
+  align-items: center;
+  background: var(--theme-transactions-header-background-color);
+  box-shadow: 0 0px 0 0 rgba(0, 0, 0, 0.25);
```

Comment:
@daniloprates why "0px" here?
Is this shadow the same for all themes?

Reply:
Removed the `px`. The `0 0 0 0` are for the scrolling state transition. The shadow is the same for all themes.